pkg/share/manila/sharebackends/nfs.go

```diff
@@ -45,11 +45,18 @@ func (NFS) BuildSource(args *BuildSourceArgs) (*v1.PersistentVolumeSource, error
 	}, nil
 }
 
-// GrantAccess to NFS share. Allows read-write access to everyone!
+// GrantAccess to NFS share. Allows read-write access to everyone by default!
 func (NFS) GrantAccess(args *GrantAccessArgs) (*shares.AccessRight, error) {
+	var accessTo string
+
+	if len(args.Options.BackendOptions.NFSShareClient) == 0 {
+		accessTo = "0.0.0.0"
+	} else {
+		accessTo = args.Options.BackendOptions.NFSShareClient
+	}
 	return shares.GrantAccess(args.Client, args.Share.ID, shares.GrantAccessOpts{
 		AccessType:  "ip",
-		AccessTo:    "0.0.0.0/0",
+		AccessTo:    accessTo,
```

Comment:
See the comment [here](https://github.com/kubernetes/cloud-provider-openstack/pull/370/files#r236597414), with `value:"default=..."` this can be simplified into

```go
func (NFS) GrantAccess(args *GrantAccessArgs) (*shares.AccessRight, error) {
	return shares.GrantAccess(args.Client, args.Share.ID, shares.GrantAccessOpts{
		AccessType:  "ip",
		AccessTo:    args.Options.BackendOptions.NFSShareClient,
		AccessLevel: "rw",
	}).Extract()
}
```

Reply:
Yup. Thanks.